my-dsl.md

```diff
@@ -2,37 +2,60 @@
 _What is the name of the language? Link the name to its webpage 
 (if appropriate)._
 
+[Mathematica/Wolfram Alpha](http://www.wolfram.com/mathematica/?source=nav)
+
 
 # Domain
 _Describe the language's domain in five words._
 
+To create mathematical figures/expressions
```

Comment:
I'd argue that Mathematica is a mathematical computation program. It does draw nice figures, but in my opinion, most people use it for computations.